### DISCUSSION TITLE
outliers in water example

### DISCUSSION THREAD

**Discussion by yufengwhy** [deepmodeling/deepmd-kit#1415] [Q&A]
1. Over 50% coords are out of the box in the dataset of water example ?  Why?
2. How should we handle this?
3. You just put the outliers to the nearest surface of the box? [code](https://github.com/deepmodeling/deepmd-kit/blob/master/source/lib/src/neighbor_list.cc#L81-L96) L89 and L96. Why?

**Comment by njzjz** [comment]
Please read _[periodic boundary conditions](https://en.wikipedia.org/wiki/Periodic_boundary_conditions)_.
